components/style/properties/properties.mako.rs

```diff
@@ -250,21 +291,24 @@ pub mod animated_properties {
             "name": "CSSWideKeyword",
             "type": "WideKeywordDeclaration",
             "doc": "A CSS-wide keyword.",
+            "copy": False,
```

Comment:
This could in theory be copy, right?

Reply:
Yes but the order of variants mean it doesn't change much to know about that.

Reply:
Fair enough